ansible/roles/cdh_common/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
     - topology.sh
 
 - name: run update-alternatives to install hadoop configuration
-  command: update-alternatives --install /etc/hadoop/conf hadoop-conf /etc/hadoop/conf.mycluster 50
+  alternatives: name=hadoop-conf link=/etc/hadoop/conf path=/etc/hadoop/conf.mycluster
```

Comment:
`alternatives` is an extra module now,
http://docs.ansible.com/ansible/alternatives_module.html#this-is-an-extras-module
and I think it is enough to use it after it becomes core module.

cf (in Japanese):
http://onigra.github.io/blog/2014/12/11/ansible-extra-and-alternatives-module/